packages/core/utils/src/convert-query-params.ts

```diff
@@ -343,6 +343,15 @@ const createTransformer = ({ getModel }: TransformerOptions) => {
     const { attributes } = schema;
 
     return Object.entries(populate).reduce((acc, [key, subPopulate]) => {
+      if (_.isString(subPopulate)) {
+        try {
+          const subPopulateAsBoolean = parseType({ type: 'boolean', value: subPopulate });
+          return { ...acc, [key]: subPopulateAsBoolean };
+        } catch {
+          // ignore
+        }
+      }
+
```

Comment:
Maybe it's adding a comment explaining why we need this new parse op. (aka "true" can be passed as a string)